appveyor.yml

```diff
@@ -18,7 +18,7 @@ install:
 # Post-install test scripts.
 test_script:
   # run tests
-  - ps: node_modules/.bin/mocha test --timeout 4000 --R
+  - ps: node_modules/.bin/gulp ; node_modules/.bin/mocha test --timeout 4000 --R
```

Comment:
I agree we would want npm scripts here.  The code in the master branch doesn't use npm scripts either, and I am trying not to change behavior in the translation to typescript PRs.  I'm concerned that perhaps there is a subtle reason why npm scripts aren't being used, and thus it could be better to add npm scripts in a follow-up PR that is independent of the translation to typescript.